docs/how-to's/ratelimiting-shared-redis.md

```diff
@@ -0,0 +1,69 @@
+# Deploying/Configuring Redis,Limitador and Rate limit policies.
```

Comment:
One comment - no issue with the re-org, but I would shy away from an apostrophe in a doc path for the sake of windows and other users.

Reply:
Yes oops forgot it would make things difficult 😅